docs/src/running-validator/validator-reqs.md

```diff
@@ -2,6 +2,13 @@
 title: Validator Requirements
 ---
 
+## Minimum SOL requirements
+
+There is no strict minimum amount of SOL required to run a validator on Solana.
+However a vote account is required, which has a rent-exempt reserve of
```

Comment:
A vote account is only required to participate in in consensus, an observer/RPC node doesn't need this of course.  I think it would be nice to make this clear here as well

Reply:
Followup incoming!